borg/archiver.py

```diff
@@ -406,64 +406,127 @@ def item_is_hardlink_master(item):
     def do_diff(self, args):
         """Diff contents of two archives"""
         def format_bytes(count):
-            if count is None:
-                return "<deleted>"
-            return format_file_size(count)
+            if count > 0:
+                sign = '+'
+            elif count < 0:
+                sign = '-'
+            else:
+                sign = ''
+
+            return '{}{}'.format(sign, format_file_size(abs(count)))
```

Comment:
maybe that functionality could be moved into format_file_size? or a separate helper function.